specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json

```diff
@@ -1943,7 +1943,6 @@
         },
         "identity": {
```

Comment:
Two questions:
1. I see idenity contains type of enum SystemAssigned. Will in the future it is maybe user assignable? If so then you may have an issue with tagging this readonly.
2. location field below. Is it type to the Azure DC? If it is, typically that is readonly as well.

Reply:
As far as I know, this rest api will throw errors when you did not specify a `identity` property, thus this should not be read only, the service is expecting a user input for this property.
As for the other questions, @zhaomuzhi can you have a look?

Reply:
Actually for 2. Here is the definition in the common types.json. Do you want to reference it over there?
`        "location": {
          "type": "string",
          "x-ms-mutability": [
            "read",
            "create"
          ],
          "description": "The geo-location where the resource lives"
        }`

Reply:
Well, seems those you are referencing here is a part of `TrackedResource`, there are a lot of other types of Resource base in the `common-types/**/types.json` file. I do not know what kind this resource is exactly, therefore for safety I think it is better to leave this problem for the service team to fix in the future, since the location is not causing problem currently.